jax/interpreters/mlir.py

```diff
@@ -674,10 +685,10 @@ def aval_to_types(aval):
 
   input_types = map(aval_to_types, jaxpr.in_avals)
   output_types = map(aval_to_types, jaxpr.out_avals)
+  num_tokens = len(effects)
```

Comment:
Does one effect always lead to one token?

Reply:
Yes.